modules/terraform/validate.go

```diff
@@ -1,10 +1,34 @@
 package terraform
 
 import (
+	// We alias Golang's native testing package to go_test to avoid naming conflicts with terratest's own testing module
+	go_test "testing"
+
 	"github.com/gruntwork-io/terratest/modules/testing"
 	"github.com/stretchr/testify/require"
 )
 
+// ValidateAllTerraformModules rutomatically finds all folders specified in RootDir that contain .tf files and runs
+// InitAndValidate in all of them.
+// Filters down to only those paths passed in ValidationOptions.IncludeDirs, if passed.
+// Excludes any folders specified in the ValidationOptions.ExcludeDirs. IncludeDirs will take precedence over ExcludeDirs
+// Use the NewValidationOptions method to pass relative paths for either of these options to have the full paths built
+// Note that go_test is an alias to Golang's native testing package created to avoid naming conflicts with Terratest's
+// own testing package
```

Comment:
NIT: Mention why we are using the native testing.T and not the terratest testing.T

Reply:
Done in 95eb0a9